src/types/zoneserver.ts

```diff
@@ -3,6 +3,7 @@ export interface Client {
   firstLoading: boolean;
   isLoading: boolean;
   isInteracting: boolean;
+  falling: number;
```

Comment:
since it's linked to character or vehicle, i would move that to client.character or vehicle